### DISCUSSION TITLE
How do I find people whom I'm following

### DISCUSSION THREAD

**Discussion by SpechtMagnus** [community/community#13187] [Other Feature Feedback, Questions, & Ideas] [answered]
Seriously, there must be a list somewhere of all the people I'm following. This driving me crazy! The only way I found to work is by clicking the tiny link at the bottom of my feed that says "ProTip! The feed shows you events from people you [follow](https://github.com/SpechtMagnus?tab=following) and repositories you [watch](https://github.com/watching) or [star](https://github.com/SpechtMagnus?tab=stars).".

**Comment by tripleo1** [comment, marked as answer]
It's on your profile page.

[https://github.com/SpechtMagnus?tab=following](https://github.com/SpechtMagnus?tab=following)

**Reply by igogo2** [reply]
You should click here on the **Your Profile** page.
![image](https://github.com/community/community/assets/6361239/36f12e2e-295b-44ed-b244-5c9e9bd7d487)

**Reply by fasakinhenry** [reply]
Alternatively, Just put `?tab=followers` or `?tab=following` at the end of your profile link.

For example:

mine is : https://github.com/fasakinhenry?tab=followers

**Comment by NeoForce0** [comment]
I see now its on the profile side ... but why not have this on the dashboard and looking at all links its not easily findable. So please place a link on the dashboard.
